wordle-web/pages/game.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <v-row justify="center" align="center">
+    <v-col cols="12" sm="8" md="6">
+        <v-card-title class="headline justify-center">
+          <h1>Wordln't</h1>
+        </v-card-title>
```

Comment:
```suggestion
        <v-card-title class="headline text-h1 justify-center">
          Wordln't
        </v-card-title>
```

Consider using the  Vuetify classes rather than HTML tags for setting heading sizes: https://vuetifyjs.com/en/styles/text-and-typography/